Training 453/app/css/normalize.css

```diff
@@ -0,0 +1,527 @@
+/*! normalize.css v1.1.2 | MIT License | git.io/normalize */
```

Comment:
be consistent on using css or scss. It's better always to use the scss preprocessor